src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+// variables
+$day-size: 100px;
+$day-font-size: 30px;
+$day-gap: 1px;
+$calendar-padding: 10px;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  font-family: Arial, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: calc($day-size * 7 + $day-gap * 6 + $calendar-padding);
+  padding-left: $calendar-padding;
+  gap: $day-gap;
+
+  &__day {
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+    font-size: $day-font-size;
+    line-height: $day-size;
+    text-align: center;
+    transition: transform 0.5s;
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
```

Comment:
Also, you should change background color
<img width="455" alt="image" src="https://github.com/user-attachments/assets/31663924-4a4c-4f8a-aa7e-b69a39c7074e">